java/src/jmri/jmrit/vsdecoder/swing/VSDControl.java

```diff
@@ -257,7 +257,7 @@ public void propertyChange(PropertyChangeEvent event) {
      */
     protected void deleteButtonPressed(ActionEvent e) {
         log.debug("({}) Delete Button Pressed", address);
-        firePropertyChange(DELETE, address, address);
+        firePropertyChange(DELETE, null, address);
```

Comment:
These values are backwards. When removing a property, the old value should be the removed item and the new value should be null.

Reply:
Thank you!